tools/clang/tools/dxopt/dxopt.cpp

```diff
@@ -62,25 +63,27 @@ bool isFileInputArg(LPCWSTR arg) {
 }
 
 static HRESULT ReadStdin(std::string &input) {
-  HANDLE hStdIn = GetStdHandle(STD_INPUT_HANDLE);
```

Comment:
No need to change for this PR, but it looks like we could dramatically simplify this tool by using the LLVM filesystem interfaces.